DancingGoat/Utils/GuidUtils.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace DancingGoat.Utils
+{
+    public class GuidUtils
+    {
+        public static Guid FromShortString(string str)
+        {
+            return Guid.ParseExact(str, "N");

```

Comment:
This method is used only once, and only has one line of code, there is no point in having a class for it when you can call that one line at the location where it is used ...